examples/02_control_flow/02_Loops.md

```diff
@@ -1,4 +1,4 @@
-#Loops
+# Loops
```

Comment:
Hey! Currently, it works without space, is not it?

Reply:
It does not - markdown headers require a space between the # and the text 🙂 you'll notice GitHub does not render it as a header before this PR.